data/crac-creation/crac-creator-csa-profiles/src/test/java/com/powsybl/openrao/data/craccreation/creator/csaprofile/craccreator/remedialaction/GroupRemedialActionTest.java

```diff
@@ -0,0 +1,134 @@
+        assertEquals(InstantKind.CURATIVE, ur3.getInstant().getKind());
+        assertEquals(UsageMethod.AVAILABLE, ur3.getUsageMethod());
+        assertEquals("RTE_AE1 (f7708112-b880-4674-98a1-b005a01a61d5) - RTE_CO1 - curative", ((OnFlowConstraint) ur3).getFlowCnec().getId());
+
+        assertNetworkActionImported(cracCreationContext, "66979f64-3c52-486c-84f7-b5439cd71765", Set.of("FFR1AA1X-FFR1AA1--1"), true, 1);
+        assertEquals("Topo 1 - PRA", cracCreationContext.getCrac().getRemedialAction("66979f64-3c52-486c-84f7-b5439cd71765").getName());
+        UsageRule ur4 = cracCreationContext.getCrac().getNetworkAction("66979f64-3c52-486c-84f7-b5439cd71765").getUsageRules().iterator().next();
+        assertEquals(InstantKind.PREVENTIVE, ur4.getInstant().getKind());
+        assertEquals(UsageMethod.AVAILABLE, ur4.getUsageMethod());
+    }
+
+    @Test
+    void importGroupedHvdcRemedialActions() {
+        CsaProfileCracCreationContext cracCreationContext = getCsaCracCreationContext("/RemedialActionGroups_HVDC.zip");
+        assertNetworkActionImported(cracCreationContext, "hdvc-200-be-de", Set.of("BBE1AA1 _generator", "DDE2AA1 _generator", "BBE1AA1  BBE4AA1  1", "DDE1AA1 _generator", "BBE2AA1 _generator", "DDE3AA1  DDE4AA1  1"), true, 1);
+        assertEquals("HDVC Action - 200 MW BE to DE", cracCreationContext.getCrac().getRemedialAction("hdvc-200-be-de").getName());
+        UsageRule ur0 = cracCreationContext.getCrac().getNetworkAction("hdvc-200-be-de").getUsageRules().iterator().next();
+        assertEquals(InstantKind.CURATIVE, ur0.getInstant().getKind());
+        assertEquals(UsageMethod.AVAILABLE, ur0.getUsageMethod());
+
+        assertNetworkActionImported(cracCreationContext, "hdvc-200-de-be", Set.of("BBE1AA1 _generator", "DDE2AA1 _generator", "BBE1AA1  BBE4AA1  1", "DDE1AA1 _generator", "BBE2AA1 _generator", "DDE3AA1  DDE4AA1  1"), true, 1);
+        assertEquals("HDVC Action - 200 MW DE to BE", cracCreationContext.getCrac().getRemedialAction("hdvc-200-de-be").getName());
+        UsageRule ur1 = cracCreationContext.getCrac().getNetworkAction("hdvc-200-be-de").getUsageRules().iterator().next();
+        assertEquals(InstantKind.CURATIVE, ur1.getInstant().getKind());
+        assertEquals(UsageMethod.AVAILABLE, ur1.getUsageMethod());
+
+        assertNetworkActionImported(cracCreationContext, "hdvc-0", Set.of("BBE1AA1 _generator", "DDE2AA1 _generator", "BBE1AA1  BBE4AA1  1", "DDE1AA1 _generator", "BBE2AA1 _generator", "DDE3AA1  DDE4AA1  1"), true, 1);
+        assertEquals("HDVC Action - 0 MW", cracCreationContext.getCrac().getRemedialAction("hdvc-0").getName());
+        UsageRule ur2 = cracCreationContext.getCrac().getNetworkAction("hdvc-0").getUsageRules().iterator().next();
+        assertEquals(InstantKind.CURATIVE, ur2.getInstant().getKind());
+        assertEquals(UsageMethod.AVAILABLE, ur2.getUsageMethod());
+    }
```

Comment:
Can you also check the elementary actions (network element + setpoint/action type) please?